references/plugins/references.go

```diff
@@ -557,25 +557,26 @@ func setDisplayFormat(format string) {
 }
 
 // GenerateReferenceDocs generates reference docs
-func (ref *MarkdownReference) GenerateReferenceDocs(ctx context.Context, c common.Args, baseRefPath string, namespace string) error {
+func (ref *MarkdownReference) GenerateReferenceDocs(ctx context.Context, c common.Args, baseRefPath string, readFrom string) error {
```

Comment:
`readFrom` seems odd.

Reply:
as it could be namespace or local filepath, i think it's not appropriate to be named as namespace any more.

Reply:
You can refer to the refactoring.

```
type Remote struct {
	DefinitionName string `json:"definitionName"`
	Namespace string `json:"namespace"`
}

type Local struct {
	Path string `json:"path"`
}

// MarkdownReference is the struct for capability information in
type MarkdownReference struct {
	Remote Remote `json:"remote"`
	Local Local `json:"local"`
	ParseReference
}
```

We would also remove `namespace` parameter from the function `GenerateReferenceDocs`.

It seems more clear. What's your idea?

Reply:
```go
type Remote struct {
	Namespace string `json:"namespace"`
}

type Local struct {
	Path string `json:"path"`
}

// MarkdownReference is the struct for capability information in
type MarkdownReference struct {
	Remote Remote `json:"remote"`
	Local Local `json:"local"`
        DefinitionName string `json:"definitionName"`
	ParseReference
}
```

I've set DefinitionName as an attribute of MarkdownReference cause it's used in both local and remote ways to generate docs.